src/components/Helpjuice/Helpjuice.tsx

```diff
@@ -90,6 +91,7 @@ export const Helpjuice: React.FC = () => {
       knowledgeBaseLink instanceof HTMLAnchorElement
     ) {
       knowledgeBaseLink.href = process.env.HELPJUICE_KNOWLEDGE_BASE_URL;
+      knowledgeBaseLink.target = '_blank';
```

Comment:
Isn't this link already `target="_blank"`?

Reply:
true